README.md

```diff
@@ -360,7 +360,7 @@ Alamofire.request("https://httpbin.org/get").validate().responseJSON { response
 
 Response Caching is handled on the system framework level by [`URLCache`](https://developer.apple.com/reference/foundation/urlcache). It provides a composite in-memory and on-disk cache and lets you manipulate the sizes of both the in-memory and on-disk portions.
 
-> By default, Alamofire leverages the shared `URLCache`. In order to customize it, see the [Session Manager Configurations](#session-manager-configurations) section.
+> By default, Alamofire leverages the shared `URLCache`. In order to customize it, see the [Session Manager Configurations](#advanced-usage) section.
```

Comment:
These should be `session-manager` instead of `advanced-usage` throughout this PR.